tests/common/snappi_tests/common_helpers.py

```diff
@@ -107,13 +107,16 @@ def get_lossless_buffer_size(host_ans):
     """
     config_facts = host_ans.config_facts(host=host_ans.hostname,
                                          source="running")['ansible_facts']
+    device_mtd = config_facts['DEVICE_METADATA']['localhost']['hwsku']
+
     is_cisco8000_platform = True if 'cisco-8000' in host_ans.facts['platform_asic'] else False
+    is_nokia_7250 = True if ('Nokia' or 'nokia') and '7250' in device_mtd else False
 
     if "BUFFER_POOL" not in list(config_facts.keys()):
         return None
 
     buffer_pools = config_facts['BUFFER_POOL']
-    profile_name = 'ingress_lossless_pool' if is_cisco8000_platform else 'egress_lossless_pool'
+    profile_name = 'ingress_lossless_pool' if (is_cisco8000_platform or is_nokia_7250) else 'egress_lossless_pool'
```

Comment:
Changes to use the `egress_lossless_pool` instead of the `ingress_lossless_pool` should apply to all Broadcom DNX based platforms.

Can this selection be made by checking for `broadcom-dnx` in the platform duthost facts?